graylog2-server/src/main/java/org/graylog2/notifications/NotificationServiceImpl.java

```diff
@@ -126,11 +153,29 @@ public boolean publishIfFirst(Notification notification) {
         try {
             save(notification);
             auditEventSender.success(AuditActor.system(nodeId), SYSTEM_NOTIFICATION_CREATE, notification.asMap());
-        } catch(ValidationException e) {
+
+            final EventDefinitionDto systemEventDefinition =
```

Comment:
this would be nicer if it's extracted into a separate method. e.g. `createSystemEvent(notifcation)`